src/axom/inlet/SphinxWriter.hpp

```diff
@@ -39,14 +39,30 @@ namespace inlet
 class SphinxWriter : public Writer
 {
 public:
+  /*!
+  *******************************************************************************
+  * \brief Supported Sphinx style options
+  *******************************************************************************
+  */
+  enum class Style
+  {
+    // FIXME: These names are not clear at all
+    Singular,  // Tables for the child Fields/Functions of each Container
+    Nested  // Table of contents for each Container, then sections for each child

```

Comment:
Perhaps `FlatTOC` and `NestedTOC`

Reply:
`flat` is better than my previous suggestion.  After speaking with @joshessman-llnl , we are gonna move forward with the exciting answer of `Flat` and `Nested`